### PR TITLE
Create Docker image for moment-proxy

### PR DESCRIPTION
**Pull request by poppoerika**
Adding local container for running momento-proxy.
The below are the instructions for how to run the momento-proxy Docker image:

You can run the `momento-proxy` container by pulling it from [Momento's](https://momentohq.com/) [docker hub registry](https://hub.docker.com/u/gomomento) with following commands.

```
docker pull gomomento/momento-proxy
docker run -d \
  -p 11211:11211 -p 9999:9999 \ 
  -e MOMENTO_AUTHENTICATION=<YOUR_MOMENTO_TOKEN> \
  gomomento/momento-proxy
```

By default, [this configuration](../../../../config/momento_proxy.toml) is used for the Momento proxy.
To set your own, please provide an env variable `CONFIG` as well as the directory where your config file is located to `-v` when running a container.

```
docker run -d \
  -p 11211:11211 -p 9999:9999 \
  -e MOMENTO_AUTHENTICATION=<YOUR_MOMENTO_TOKEN> \
  -e CONFIG=<YOUR_CONFIG_FILE> \
  -v /your/path/to/config/dir:/app/config \
  gomomento/momento-proxy
```

### CODE REVIEW

**Comment by poppoerika** [comment]
@brayniac @eaddingtonwhite @danielamiao
Could you review this pr? Thank you!

File: src/rust/proxy/momento/Cargo.toml [hunk 1]
```diff
@@ -21,3 +21,17 @@ protocol-memcache = { path = "../../protocol/memcache" }
...
+license-file = ["../../../../LICENSE", "0"]
+depends = "$auto"
+section = "web"
+priority = "optional"
+assets = [
+	["target/release/momento_proxy", "usr/bin/momento_proxy", "755"],
+	["../../../../config/momento_proxy.toml", "etc/momento_proxy/config.toml.example", "644"],	
+]
+maintainer-scripts = "debian/"
+systemd-units = { enable = false }
```
Comment (eaddingtonwhite) [inline src/rust/proxy/momento/Cargo.toml]: @brayniac I think maybe I messed this up in merging over changes from latest branch. Not sure why this is showing up in diff do you expect this change to be here.

File: src/rust/proxy/momento/debian/service [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=a proxy to access Momento cache(s) over memcache protocol
+
+[Service]
+EnvironmentFile=/etc/momento_proxy/environment
+ExecStart=/usr/bin/momento_proxy /etc/momento_proxy/config.toml
+
+[Install]
+WantedBy=multi-user.target
```
Comment (eaddingtonwhite) [inline src/rust/proxy/momento/debian/service]: @brayniac Same here Not sure why this is showing up in diff do you expect this change to be here.

Comment (brayniac) [reply src/rust/proxy/momento/debian/service]: This should not be here - you should be able to delete the `debian ` folder

File: src/rust/proxy/momento/Cargo.toml (hunk 1 above)
Comment (brayniac) [reply src/rust/proxy/momento/Cargo.toml]: This should not be here - I think I had reverted back to `c6a32f6` and rewrote the history.

**Review by brayniac — COMMENTED** [review]
Other than the debian dpkg stuff that got pulled back in, this looks good.

**Comment by poppoerika** [comment]
@brayniac - Removed the `debian` package.

**Review by brayniac — APPROVED** [review]
Looks good. Thanks for contributing this!

**Pull request merged**